### ISSUE TITLE
recursive brace layers 

### ISSUE THREAD

**Issue by alexeiva** [googlefonts/fontmake#770]
> Yes, you don't need to expand the brace layers into font-wide masters (which loses all the benefits of sparse intermediate masters).
> You only need to do that for any composite glyphs whose components' base glyph use brace layers, as well as for all the other components' base glyphs which do not have brace layers.
> 
> For example, for your "NJ" composite glyph, where the "N" glyph has brace layers, but "NJ" and "J" do not:
> 1) in the "NJ" composite glyph, you make a copy of the master layers and rename them to the brace layers' name of the "N" glyph;
> 2) in the "J" outline glyph, similarly you duplicate the master layers and rename them as the brace layers, then you right-click on  "Re-interpolate" to get the correct interpolated outline (and do not retouch it afterwards).
> 
> I tried this workaround on a couple of affected composite glyphs in BigShoulders font and I can confirm that it is working as expected when building static instances or variable font using fontmake.

_Originally posted by @anthrotype in https://github.com/googlefonts/fontmake/issues/569#issuecomment-522219263_

This is related to the mentioned issue, my current workflow:
1. I have brace layers in `I`, 
2. I create brace layers for `Í` `Ĭ`, etc.
3. fontmake needs brace layers for `acutecomb.case`, `brevecomb.case`, etc.
4. fontmake needs brace layers for all glyphs that have `acutecomb.case`, i.e. `ÁÉŃŔŚÝ`, 
5. As a result nearly all component glyphs will have brace layers. 

Any thoughts on this?

**Comment by alexeiva** [comment]
more details in https://github.com/googlefonts/youtube-sans-mono/issues/19

**Comment by alexeiva** [comment]
I resolved the metrics discrepancy issue by adding 577 brace layers. Total number of glyphs is 979.
Used @yanone 's [script](https://github.com/yanone/Yanone-GlyphsApp-Scripts/blob/master/Interpolation/Create%20Missing%20Brace%20Layers%20in%20Components.py), and manually added missing glyphs based on gftools builder output. In the end I added all components with auto-alignment. 

For example here is a list of uppercase glyphs with brace layers:
```AÁĂẮẶẰẲẴÂẤẬẦẨẪȀÄẠÀẢȂĀĄÅǺÃÆǼCĆČÇĈĊDĎEÉĔĚÊẾỆỀỂỄȄËĖẸÈẺȆĒĘẼGĞǦĜĢĠHĦĤIÍĬÎȈÏİỊÌỈȊĪĮĨJĴKĶLĹĽĻĿŁNŃŇŅÑOÓŎÔỐỘỒỔỖȌÖȪȰỌÒỎƠỚỢỜỞỠŐȎŌǪØǾÕȬŒRŔŘŖȐȒSŚŠŞŜȘTŦŤŢȚUÚŬÛȔÜỤÙỦƯỨỰỪỬỮŰȖŪŲŮŨWẂŴẄẀYÝŶŸỴỲỶȲỸZŹŽŻ```
